### PR TITLE
Plugins plans: Update to show mobile view

### LINKED ISSUE #67961: Marketplace: Plan information step for other screen sizes
Follow-up issue for #67574

See [comment](https://github.com/Automattic/wp-calypso/issues/67574#issuecomment-1249319530) for more details.

Acceptance criteria:

- [ ]  Has the same function as the desktop version for other screen sizes

### PR DESCRIPTION
**Pull request by lupus2k**
#### Proposed Changes

Following #67891, needs mobile view for plugins plans.

#### Testing Instructions

* Go to `/plugins/plans/:site` using mobile view
* Plans should look like the `After` screenshot.

#### Screenshot
| Before | After | 
| --- | --- |
| image | image |

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Fixes #67961

### CODE REVIEW

**Comment by lupus2k** [comment]
Hey, @vinimotaa, following [this](https://github.com/Automattic/wp-calypso/issues/67574#issuecomment-1249319530), I updated the `/plugins/plans` using the mobile view.

Screenshots [here](https://user-images.githubusercontent.com/402286/191295739-e4565f63-101a-4d5c-a569-960e2315d4b2.png) (showing 2 plans) and [here](https://user-images.githubusercontent.com/402286/191295698-50a94155-8a35-4c91-9f9e-b60b93219a98.png) (showing 3).

Adding margin to `.plans-features-main` and `.plugin-plans-header` will look like [this](https://user-images.githubusercontent.com/402286/191296490-3e56f239-4763-4c05-8cc9-2816c1ee93fe.png), does it looks good for you?

When it shows only 2 plans, it shows a table under the plans, do you think we have to have it?

Thanks

**Comment by lupus2k** [comment]
I updated the CSS.
| w/ Free plan | w/ Business plan |
| --- | --- |
| image | image |

File: client/my-sites/plugins/plans/index.tsx [hunk 1]
```diff
@@ -44,9 +45,20 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
...
+	useEffect( () => {
+		const onResize = () => {
+			setIsDesktopDevice( isDesktop() || false );
+		};
+
+		onResize();
+		window.addEventListener( 'resize', onResize );
+
+		return () => window.removeEventListener( 'resize', onResize );
+	}, [] );
```
Comment (lsl) [inline client/my-sites/plugins/plans/index.tsx:57]: There's a hook for this. https://github.com/Automattic/wp-calypso/blob/9e36d03760163ecfd883239ae6e8b17be3d4523f/client/my-sites/plans/jetpack-plans/product-grid/index.tsx#L115

File: client/my-sites/plugins/plans/style.scss [hunk 2]
```diff
@@ -6,3 +6,17 @@
...
+	@include breakpoint-deprecated( "<660px" ) {
+		margin: 0 24px !important;
+
+		.formatted-header {
+			display: none;
+		}
+
+		.plan-features__table {
+			display: none !important;
+		}
```
Comment (lsl) [inline client/my-sites/plugins/plans/style.scss:20]: Plans table already handles breakpoints under /start - why do we need these extra display overrides here? Is there something specific to `isInSignup` that should be on the `isReskinned` flag instead?

Screenshot 2022-09-21 at 14-57-47 Create a site — WordPress com

File: client/my-sites/plugins/plans/index.tsx (hunk 1 above)
Comment (lupus2k) [reply client/my-sites/plugins/plans/index.tsx:57]: Thanks, I searched and didn't found it.

**Comment by lupus2k** [comment]
Closed because it's fixed in #68065

File: client/my-sites/plugins/plans/index.tsx (hunk 1 above)
Comment (lsl) [reply client/my-sites/plugins/plans/index.tsx:57]: Calypso is huge, reinventing the wheel is going to happen sometimes :slightly_smiling_face:

**Pull request closed without merging**